src/LayerEditor/ui/data/tools/selector.py

```diff
@@ -0,0 +1,23 @@
+
+
+class Selector:
+    """ Used for selecting one item(value) from list.
```

Comment:
The value becomes property in later pull request. Keeping reference to the list is not possible since it exists only temporarily. Index also isnt possible to make since sometimes(now always) we worked with dictionaries. Ids also cannot be used for this purpose.